conftest.py

```diff
@@ -104,3 +106,21 @@ def test_post_teardown():
 
     # Make sure we revert everything we might have changed to prefixers.
     amo.urlresolvers.clean_url_prefixes()
+
+
+@pytest.fixture
+def admin_group(db):
```

Comment:
Summary of how it works: unless the fixture is `autouse`, it's only used if it's needed in a test. To "require" it in a test, simply add a parameter to the test function which has the name of the fixture.
So as you can see below, the fixture `mozilla_user` requires the `admin_group` fixture. And the `test_watch_source` test uses the `mozilla_user`.
The `admin_group` uses the `db` fixture which is provided by [pytest-django](http://pytest-django.readthedocs.org/en/latest/helpers.html#pytest.mark.django_db), which allows the test to access the database.

When defining a fixture, you can change its [scope](http://pytest.org/latest/fixture.html#sharing-a-fixture-across-tests-in-a-module-or-class-session): by default it's `function` which means the fixture is executed before each test function, but it can be module or class.
The `db` fixture has the `function` scope, meaning that it's executed before each test. It acts just like the django tests which [reverts the data](http://pytest-django.readthedocs.org/en/latest/helpers.html#pytest.mark.django_db) once the test has run.